src/components/weather/forecast/ForecastByHourList.tsx

```diff
@@ -249,9 +250,12 @@ const ForecastByHourList: React.FC<ForecastByHourListProps> = ({
               style={[
                 styles.row,
                 styles.listContainer,
-                styles.justifySpaceBetween,
+                styles.maxWidth,
+                styles.justifyContentCenter,
```

Comment:
Maybe we should keep both styles (content-center, space-between) and use them according to the config variable? This looks a bit different from the current Finnish implementation with the current configuration (day duration not excluded).

Reply:
we'll make changes to fi version. Panel looks awful with landscape or tablet screens. Added more margin to make it less compact.